BT-Tracking/Services/Reports/ReportService.swift

```diff
@@ -38,131 +36,193 @@ protocol ReportServicing: class {
 
 class ReportService: ReportServicing {
 
-    private var folderPattern: String = "exposure"
-    private var filePattern: String = "exposure.json"
+    var healthAuthority = "cz.covid19cz.erouska.dev"
+
     private var timeout: TimeInterval = 30
 
+    private let uploadURL = URL(string: "https://exposure-i5jzq6zlxq-ew.a.run.app/v1/publish")!
+
+    private let downloadBaseURL = URL(string: "https://storage.googleapis.com/exposure-notification-export-ejjud/")!
```

Comment:
Tyto URL by to pak chtěl vytáhnout na nějaký společný místo, aby tady nevysely v kódu jako stringy.

Reply:
Jo chci potom udelat Config ktery se bude predavat